src/_mixins.scss

```diff
@@ -66,4 +91,57 @@ $toplevel-border-color: rgba(0, 0, 0, 0.3);
     }
 }
 
+// elements with outset relief style like headerbars, buttons, checkboxes, etc
+%outset {
+    background-image:
+        linear-gradient(
+            to bottom,
+            #{'alpha(@highlight_color, 0.2)'},
+            rgba(255, 255, 255, 0)
+        );
+    box-shadow:
+        inset 0 1px 0 0 #{'alpha(@highlight_color, 0.3)'},
+        inset 1px 0 0 0 #{'alpha(@highlight_color, 0.07)'},
+        inset -1px 0 0 0 #{'alpha(@highlight_color, 0.07)'},
+        inset 0 -1px 0 0 #{'alpha(@highlight_color, 0.2)'},
+        0 0 0 1px rgba(0, 0, 0, 0.15),
+        0 1px 2px rgba(0, 0, 0, 0.2);
+
+    &.titlebar {
```

Comment:
I'm not sure if it makes sense to keep this here so that we always sync these values or if this should go in widgets since it's specific to that one widget

Reply:
I _think_ it would make more sense in widgets since it's specific to a widget and not generally useful.